templates/layout.html

```diff
@@ -5,6 +5,28 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% if post %}
```

Comment:
Does this need `if post && self`?

Reply:
I don't think so. The only block using `self` is title and is wrapped. Also isn't `self` implicit?

Reply:
Oh I see its outside of this block. Read it wrong.